feature/onboarding/permissions/src/main/kotlin/app/k9mail/feature/onboarding/permissions/ui/PermissionBox.kt

```diff
@@ -0,0 +1,172 @@
+package app.k9mail.feature.onboarding.permissions.ui
+
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import app.k9mail.core.ui.compose.designsystem.atom.Icon
+import app.k9mail.core.ui.compose.designsystem.atom.button.Button
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextBody2
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextHeadline6
+import app.k9mail.core.ui.compose.theme.Icons
+import app.k9mail.core.ui.compose.theme.K9Theme
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.feature.onboarding.permissions.R
+import app.k9mail.feature.onboarding.permissions.ui.PermissionsContract.UiPermissionState
+
+private val MAX_WIDTH = 500.dp
+private val PERMISSION_ICON_SIZE = 32.dp
+private val PERMISSION_STATE_ICON_SIZE = 18.dp
+private val PERMISSION_STATE_ICON_OFFSET_X = 8.dp
+private val PERMISSION_STATE_ICON_OFFSET_Y = 5.dp
```

Comment:
This values should be derived from the `MainTheme.spacings` and `MainTheme.sizes`.  Doing so would allow to tweak values later for different screen sizes.

I guess the 5 dp offset is to avoid issues with overlaping of the icons, right?

Reply:
The values were chosen so the result is visually acceptable. The offsets should probably depend on the visual edge of an icon. I changed this to a more generic approach in 9b1d649705fcfbe5ac923e3452b199a85ad55a55